macro/run_simScript.py

```diff
@@ -380,6 +383,23 @@
  P6gen.SetMom(50.*u.GeV)
  P6gen.SetTarget("gamma/mu+","n0") # default "gamma/mu-","p+"
  primGen.AddGenerator(P6gen)
+
+# -----EvtCalc--------------------------------------
+if simEngine == "EvtCalc":
+ print(f'Opening input file for conversion: {inputFile}')
+ if not os.path.isfile(inputFile): raise FileNotFoundError("EvtCalc: input .dat file does not exist")
+ primGen.SetTarget(0., 0.)
+ inputROOTFile = (convertEvtCalc.convert_file(infile=inputFile, outdir=options.outputDir))
+ print('Input root file: ', inputROOTFile)
+ print(f'Opening input file for EvtCalc generator: {inputROOTFile}')
+ ut.checkFileExists(inputROOTFile)
```

Comment:
Would be better to do the conversion in a separate utility, and provide the ROOT file as input to run_simScript.py

Reply:
I'd prefer not having this is `run_simScript.py`. A nicer solution would be to add a small `if __name__ == "__main__":` section to `convertEvent.py` to allow it being called directly, allowing us to do the conversion for now.

And then in `run_simScript.py` we can use the output from that script.

Reply:
Solving for now. To be addressed later with a dedicated issue (interfacing EvtCalc sampling code).

Reply:
Will look into this in separate issue #535